### ISSUE TITLE
unknown authentication type: 10

### ISSUE THREAD

**Issue by morganchristiansson** [timescale/outflux#92]
I just tried it out and I get error... I also realised last release is from 2019 .. and last commit 2 years ago..
```
./outflux schema-transfer database123
2022/01/06 07:20:17 Selected input database: database123
2022/01/06 07:20:17 Overriding PG environment variables for connection with: sslmode=disable
2022/01/06 07:20:17 could not open connections to input and output database
could not open connection to TimescaleDB Server
unknown authentication type: 10
```

**Comment by Hitesh-Agrawal** [comment]
@morganchristiansson I contacted timescale support about this and the issue was with the authentication mechanism in the timescale cloud db. They suggested following solution and it worked for me
"Could you try to set up the default authentication type in Timescale Cloud from 'scram' to 'md5'?
 In order to do this, from the Cloud portal, open the Operations tab, and then click on Reset Password. After that, you would want to select the 'Authentication type' dropdown to MD5 "

**Comment by VIEWVIEWVIEW** [comment]
In case you're selfhosting: https://stackoverflow.com/a/64336274

As @Hitesh-Agrawal already stated: the tool doesn't support the more modern (and now default) ``scram-sha-256`` authentication mechanism.

**Comment by dpooley** [comment]
If you [edit go.mod](https://github.com/dpooley/outflux/commit/7c332e8a25466ca19f48a24274b60b876d24b53b) and build `outflux` with pgx v3.4.0+ then `scram-sha-256` authentication is [supported](https://github.com/jackc/pgx/commit/5044e8473ad948114b6cb63f6f30f94fc7834667), and appears to work OK (from my limited testing).

**Comment by JamesGuthrie** [comment]
Fixed in #96
